src/main/scala/cromwell/util/EnhancedExponentialBackoff.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.util
+
+import com.google.api.client.util.ExponentialBackOff
+
+class EnhancedExponentialBackoff(enhancedBuilder: EnhancedExponentialBackoff.Builder) extends ExponentialBackOff(enhancedBuilder) {
+  private var firstCall: Boolean = true
```

Comment:
I'm not sure I see how you would do that with and ADT while keeping using the existing `nextBackoffMillis` code from the Java class ?